rascal-lsp/src/main/rascal/lsp/lang/rascal/lsp/IDECheckerWrapper.rsc

```diff
@@ -82,9 +97,13 @@ map[loc, set[Message]] checkFile(loc l, set[loc] workspaceFolders, start[Module]
             checkedForImports += currentSrc;
             checkForImports -= tree;
         }
-        return true;
+        return {};
     }, totalWork=1);
 
+    if ({} !:= parseMsgs) {
+        return toMap(parseMsgs);
```

Comment:
why not make `parseMsgs` a map? it looks like the code is only building a rel with a single entry?